tests/src/test/scala/org/http4s/StaticFileSpec.scala

```diff
@@ -66,6 +86,21 @@ class StaticFileSpec extends Http4sSpec {
       response.map(_.status) must beSome(NotModified)
     }
 
+    "Send modified files when either the ETag or last modified date do not match" in {
```

Comment:
Isn't this testing when the Etag doesn't match, and it's not tested when the If-Modified-Since doesn't match?  The "either" implies two tests to me.

Reply:
Yes, you're correct, there should be two tests.